docs/rollover.md

```diff
@@ -0,0 +1,47 @@
+# Rollover
+
+Each year we close the current cycles courses and open the new cycle in a
+process we call Rollover.
+
+This involves copying existing providers and courses to new records to allow the
+providers to update any details and then switching the API over to a new
+'recruitment cycle' which in turn releases the new courses on Find & Apply.
+
+In addition to this, providers also request places for courses that have numbers
+of applicants limited as they are generally over supplied.
+This process is called Allocations.
+
+This document lists the changes needed to be made to the TTAPI codebase and the
+timings for these changes. There is separate documentation for Publish [here].
+
+## Testing the Rollover process
+
+This should happen every year in good time to allow for any code
+updates/refactoring work.
+
+1. Ensure that **testing environments** are set up for TTAPI and Publish.
+2. Begin a 'test' Rollover (see **'On Rollover launch date'**)
+3. Test v1, v2, v3 and public v1 endpoints.
```

Comment:
```suggestion
3. Test v2, v3 and public v1 endpoints.
```

We can probably drop v1 @gpeng

Reply:
Worth reminding the reader that `v1` is for UCAS?

```suggestion
3. Test v2, v3 and public v1 endpoints (v1 is the original UCAS API and soon to be deprecated).
```

Reply:
Updated